langwatch/src/hooks/useOrganizationTeamProject.ts

```diff
@@ -115,11 +115,11 @@ export const useOrganizationTeamProject = (
   const organization = teamsMatchingSlug?.[0]
     ? teamsMatchingSlug?.[0].organization
     : projectsTeamsOrganizationsMatchingSlug?.[0]
-    ? projectsTeamsOrganizationsMatchingSlug?.[0].organization
-    : organizations.data
-    ? organizations.data.find((org) => org.id == localStorageOrganizationId) ??
-      organizations.data[0]
-    : undefined;
+      ? projectsTeamsOrganizationsMatchingSlug?.[0].organization
+      : organizations.data
+      ? organizations.data.find((org) => org.id == localStorageOrganizationId) ??
+        organizations.data[0]
+      : undefined;
```

Comment:
oh gosh, this isn't awesome to read.

Reply:
I know, i reallllly dislike this kind of js, but i didn't write it so I don't want to touch it 😅